tox.ini

```diff
@@ -24,6 +25,7 @@ deps =
     django20: django==2.0.*
     django21: django==2.1.*
     django22: django==2.2.*
+    django30: django>=3.0a1,<3.1
```

Comment:
Try to keep the same style

like `django==3.*`

Reply:
Unfortunately, `django==3.0.*` won't match alpha/beta pre-releases, so it's necessary to use this style until the final version is released.